docs/introduction/basic_usage.rst

```diff
@@ -2,11 +2,17 @@
 Basic usage
 ###########
 
-Create a django CMS page to hook the Aldryn Jobs application into. In its *Advanced settings*,
-set its ``Application`` to *Jobs*, and use the provided, default *Jobs / aldryn_jobs* in
-``Application configurations`` before saving.
+Create a django CMS page to hook the Aldryn Jobs application into. In its
+*Advanced settings*, set its ``Application`` to *Jobs*, and use the provided,
+default *Jobs / aldryn_jobs* in ``Application configurations`` before saving.
 
-The page is now a landing page for job openings; it will be empty until you create some.
+The page is now a landing page for job openings; it will be empty until you
+create some.
 
-The behaviour of the Jobs system should be largely self-explanatory, but the :doc:`tutorial for
-users </user/index>` will guide you through some basic steps if necessary.
+Also in *Advanced settings*, you may choose to attach one of two menus as a
+sub-menu to the page in the site navigation: the ``Job Categories Menu`` or
```

Comment:
If "Job Categories Menu" is a field value, it should be *emphasised* - http://docs.django-cms.org/en/develop/contributing/contributing.html#inline-markup